### ISSUE TITLE
Feature Request: add skipped tests to junit reports

### ISSUE THREAD

**Issue by lyndsey-ferguson** [lyndsey-ferguson/fastlane-plugin-test_center#92]
### Feature Request

Feature Request: add skipped tests to junit reports.

#### Motivation Behind Feature

Xcode provides the option to mark a test as "Skipped". These tests do not show up in the junit report and developers do not get a sense of the impact of disabled tests. It would help them if skipped tests were added to the junit file.

#### Feature Description

A Junit Xml file can contain testcases that have been skipped. This is done by adding the element with a <skipped> child element with a message attribute. For example:

```
<testcase classname="tst_XmlPatterns" name="initTestCase" time="0.000">
  <skipped message="tst_xmlpatterns.cpp:120 :: [] Device &#x9;IP:169.254.0.13&#xA;The command line tool (/usr/lib/qt5/bin/xmlpatterns) could not be run, possibly because Qt was incompletely built or installed. No tests will be run."/>
</testcase>
```

#### Alternatives or Workarounds

N/A

**Comment by lyndsey-ferguson** [comment]
We are already collecting the skipped tests. We can, once the runner is finished, go through each testable and update the junit report(s) with this testcase/skipped elements.

**Comment by lyndsey-ferguson** [comment]
Fixed in [v3.8.0](https://github.com/lyndsey-ferguson/fastlane-plugin-test_center/releases/tag/v3.8.0)
